### ISSUE TITLE
[WebProfiler][Twig] Console error in performance panel

### ISSUE THREAD

**Issue by sabruss** [symfony/symfony#33403]
**Symfony version(s) affected**: 4.3.4

**Description**  
i've got an error when profiling performance panel, timeline doesn't work at all

![image](https://user-images.githubusercontent.com/17797150/64053570-3ad98900-cb83-11e9-981f-ce90a969021e.png)

It appear that the starttime is  `-0.1` 
And the issues is caused by the autoescape 'JS'

**How to reproduce**  

`{% autoescape 'js' %}
    {{ "%F"|format(-0.1) }}
{% endautoescape %}`

in a twig template

**Comment by pifaace** [comment]
Hi @sabruss 
i did a simple twig page with your line 
`{% autoescape 'js' %} {{ "%F"|format(-0.1) }} {% endautoescape %}`
And all is ok on my screen, did you do extra steps ?

here is my code and screens
http://zupimages.net/viewer.php?id=19/36/qp4p.png
http://zupimages.net/viewer.php?id=19/36/62h2.png
http://zupimages.net/viewer.php?id=19/36/dm32.png

**Comment by sabruss** [comment]
I don't know why my profiler get me -0.1 on starttime  on kernel.request, but in that case, it cause the profiler to fail because of the autoescape js that convert `-` as `\u002D`, then the js is invalid.
I think it's an isolated case.

i didn't do anything wrong, just checking performance and saw that issue.

**Comment by pifaace** [comment]
I mean, I cant reproduce the bug, just with 
`{% autoescape 'js' %} {{ "%F"|format(-0.1) }} {% endautoescape %}`
Maybe, there is another issue in your JSON, can you past a part of it ?

**Comment by sabruss** [comment]
When i unserialize the profiler's file at kernel.request i got a start a -0.1

the JS is correct it just seem that the collector fail at collecting the start then it fail on escaping data on js that cause that error on profiler

![image](https://user-images.githubusercontent.com/17797150/64274738-4ee70700-cf44-11e9-9e27-1dc9b85095f8.png)

**Comment by xabbuh** [comment]
Could you create a small example application that allows to reproduce the issue?

**Comment by sabruss** [comment]
Sorry i can't reproduce the issue, it appear only once.

**Comment by xabbuh** [comment]
Thank you for the feedback. Let's close for now then. We can reopen in the future if it happens again and we are able to reproduce.
